src/__tests__/incidents/view/ViewIncident.test.tsx

```diff
@@ -1,80 +1,73 @@
-import * as ButtonBarProvider from '../../../page-header/button-toolbar/ButtonBarProvider'
-import * as titleUtil from '../../../page-header/title/TitleContext'
+import { ButtonBarProvider } from '../../../page-header/button-toolbar/ButtonBarProvider'
+import { TitleProvider } from '../../../page-header/title/TitleContext'
 import IncidentRepository from '../../../shared/db/IncidentRepository'
 import Incident from '../../../shared/model/Incident'
 import Permissions from '../../../shared/model/Permissions'
 import { RootState } from '../../../shared/store'
 
-const { TitleProvider } = titleUtil
 const mockStore = createMockStore<RootState, any>([thunk])
 
-describe('View Incident', () => {
-  const setup = async (permissions: Permissions[]) => {
-    jest.resetAllMocks()
-    jest.spyOn(titleUtil, 'useUpdateTitle').mockImplementation(() => jest.fn())
-    jest.spyOn(breadcrumbUtil, 'default')
-    jest.spyOn(IncidentRepository, 'find').mockResolvedValue({
-      id: '1234',
-      date: new Date().toISOString(),
-      code: 'some code',
-      reportedOn: new Date().toISOString(),
-    } as Incident)
-    const history = createMemoryHistory()
-    history.push(`/incidents/1234`)
+const setup = (permissions: Permissions[], id: string | undefined) => {
+  jest.resetAllMocks()
+  jest.spyOn(breadcrumbUtil, 'default')
+  jest.spyOn(IncidentRepository, 'find').mockResolvedValue({
+    id,
+    date: new Date().toISOString(),
+    code: 'some code',
+    reportedOn: new Date().toISOString(),
+  } as Incident)
 
-    const store = mockStore({
-      user: {
-        permissions,
-      },
-    } as any)
+  const history = createMemoryHistory({ initialEntries: [`/incidents/${id}`] })
+  const store = mockStore({
+    user: {
+      permissions,
+    },
+  } as any)
 
-    let wrapper: any
-    await act(async () => {
-      wrapper = await mount(
-        <ButtonBarProvider.ButtonBarProvider>
-          <Provider store={store}>
-            <Router history={history}>
-              <Route path="/incidents/:id">
-                <TitleProvider>
-                  <ViewIncident />
-                </TitleProvider>
-              </Route>
-            </Router>
-          </Provider>
-        </ButtonBarProvider.ButtonBarProvider>,
-      )
-    })
-    wrapper.update()
-    return { wrapper: wrapper as ReactWrapper, history }
+  return {
+    history,
+    ...render(
+      <ButtonBarProvider>
+        <Provider store={store}>
+          <Router history={history}>
+            <Route path="/incidents/:id">
+              <TitleProvider>
+                <ViewIncident />
+              </TitleProvider>
+            </Route>
+          </Router>
+        </Provider>
+      </ButtonBarProvider>,
+    ),
   }
+}
 
-  it('should set the breadcrumbs properly', async () => {
-    await setup([Permissions.ViewIncident])
+it('smoke tests ViewIncidentDetails when there are no Permissions', async () => {
```

Comment:
`ViewIncidentsDetails` is an implementation detail

I think the test description should be `does not display incident details if there are no permissions`

Reply:
@emma-r-slight you resolve this one too?